lib/Formatter.php

```diff
@@ -21,14 +24,32 @@
 
 namespace OCA\UserUsageReport;
 
+use OC\AppFramework\Http\Request;
 use OCP\Files\FileInfo;
+use OCP\IRequest;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 trait Formatter {
 
 	protected $timestamp;
 
+	protected $userAgents = [
+		'IE' => 				Request::USER_AGENT_IE,
+		'Edge' => 				Request::USER_AGENT_MS_EDGE,
+		'Chrome' => 			Request::USER_AGENT_CHROME,
+		'Firefox' => 			Request::USER_AGENT_FIREFOX,
+		'Safari' => 			Request::USER_AGENT_SAFARI,
+		'Chrome for Android' => Request::USER_AGENT_ANDROID_MOBILE_CHROME,
```

Comment:
^ we should look into moving those to IRequest, too, and perhaps also offer an array, maybe with additional information as type ("browser", "desktop", "other-app"? at least DAV app needs browsers). That way, changes only stay in one place and we don't need to use private API in this app.